reference/price_lists.v3.yml

```diff
@@ -2,179 +2,86 @@ swagger: '2.0'
 
     **ELSE IF** `channel` has a `default price list`:
-
-    * Use this price list -- any prices not found fall back to the catalog price
-    (or in the case of multi-currency, auto-converted prices)
-
+    * Use this price list -- any prices not found fall back to the catalog price (or in the case of multi-currency, auto-converted prices)
 
     **ELSE**:
+    * Fall back to the catalog price (or in the case of multi-currency, auto-converted prices)
 
-    * Fall back to the catalog price (or in the case of multi-currency,
-    auto-converted prices)
-
-
-    **Note:** Price Lists cannot be assigned to a customer group that has custom
-    group discounts -- the customer group discounts must be deleted first.
-
+    **Note:** Price Lists cannot be assigned to a customer group that has custom group discounts -- the customer group discounts must be deleted first.
 
     ## Usage notes
-
-    - Price Lists cannot be assigned to a customer group that has custom group
-    discounts -- the customer group discounts must be deleted first.
-
-    - Bulk pricing Tiers may additionally be associated with a price record to
-    indicate different pricing as the quantity in cart increases.
-
-    - If a variant has a `Price Record` any existing product-level bulk pricing
-    will not apply in the cart. For variants without `Price Records`, any
-    existing product bulk pricing will apply.
-
-    - Price Lists Records accepts bulk upsert. Only one [Bulk
-    upsert](https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-records/setpricelistrecordcollection)
-    can done at a time. Running more than one in parallel on the **same store**
-    will cause a 429 error and the request will fail.
-
-    - There are no direct webhooks available for Price Lists. Since Price Lists
-    directly relate to products, webhooks related to products will fire for
-    corresponding changes such as pricing.
-
+    - Price Lists cannot be assigned to a customer group that has custom group discounts -- the customer group discounts must be deleted first.
+    - Bulk pricing Tiers may additionally be associated with a price record to indicate different pricing as the quantity in cart increases.
+    - If a variant has a `Price Record` any existing product-level bulk pricing will not apply in the cart. For variants without `Price Records`, any existing product bulk pricing will apply.
+    - Price Lists Records accepts bulk upsert. Only one [Bulk upsert](https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-records/setpricelistrecordcollection) can done at a time. Running more than one in parallel on the **same store** will cause a 429 error and the request will fail.
+    - There are no direct webhooks available for Price Lists. Since Price Lists directly relate to products, webhooks related to products will fire for corresponding changes such as pricing.
```

Comment:
There are no webhooks at all for price lists (we have an epic to do this at some point)
If catalog prices are updated, product webhooks fire.